### PR TITLE
update information-schema/information-schema-tiflash-replica.md

### PR DESCRIPTION
**Pull request by hehechen**
Signed-off-by: hehechen <awd123456sss@gmail.com>

### What is changed, added or deleted? (Required)
update information-schema/information-schema-tiflash-replica.md

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: information-schema/information-schema-tiflash-replica.md [hunk 1]
```diff
@@ -29,3 +29,11 @@ DESC tiflash_replica;
 +-----------------+-------------+------+------+---------+-------+
 8 rows in set (0.01 sec)
 ```
+`TABLE_SCHEMA`：表所属的数据库的名称。
+`TABLE_NAME`：表的名称。
+`TABLE_ID`：表的内部 ID，该 ID 在一个 TiDB 集群内部唯一。
+`REPLICA_COUNT`：TiFlash 副本数。
+`LOCATION_LABELS`：设置 TiFlash 副本时设置的 LocationLabelList。
+`AVAILABLE`：是否会在 TiFlash 上做查询。如果为 1 ，此时 TiDB 优化器会考虑将查询下推到 TiFlash；如果为 0 TiDB 就不会下推查询到 TIFlash。
```
Comment (qiancai) [inline information-schema/information-schema-tiflash-replica.md:37]:
```suggestion
`AVAILABLE`：是否会在 TiFlash 上做查询。如果为 1 ，TiDB 优化器将依据查询代价智能选择下推查询到 TiKV 或 TiFlash；如果为 0，TiDB 将不会下推查询到 TiFlash。
```

File: information-schema/information-schema-tiflash-replica.md [hunk 2]
```diff
@@ -29,3 +29,11 @@ DESC tiflash_replica;
...
+`PROGRESS`：TiFlash 副本同步进度，精度为分钟级。如果 `PROGRESS` 小于 1，表示 TiFlash 副本落后 TiKV 较多，查询很可能会因为等待数据同步超时而失败。
```
Comment (qiancai) [inline information-schema/information-schema-tiflash-replica.md:38]: 如果 `PROGRESS` 为 1，代表什么呢

File: information-schema/information-schema-tiflash-replica.md [hunk 3]
```diff
@@ -29,3 +29,11 @@ DESC tiflash_replica;
...
+`TABLE_MODE`：TiFlash 是否开启 FastScan。 Nomal 表示禁用 FastScan，Fast 表示开启 FashScan。
```
Comment (qiancai) [inline information-schema/information-schema-tiflash-replica.md:39]:
```suggestion
`TABLE_MODE`：TiFlash 是否开启 FastScan。 Normal 表示禁用 FastScan，Fast 表示开启 FashScan。
```

File: information-schema/information-schema-tiflash-replica.md (hunk 2 above)
Comment (hehechen) [reply information-schema/information-schema-tiflash-replica.md:38]: > 如果 `PROGRESS` 为 1，代表什么呢

就表示 TiFlash 副本与 TiKV 比较接近，查询大概率不会等待数据同步超时。

File: information-schema/information-schema-tiflash-replica.md [hunk 4]
```diff
@@ -29,3 +29,11 @@ DESC tiflash_replica;
...
 8 rows in set (0.01 sec)
 ```
+`TABLE_SCHEMA`：表所属的数据库的名称。
+`TABLE_NAME`：表的名称。
+`TABLE_ID`：表的内部 ID，该 ID 在一个 TiDB 集群内部唯一。
+`REPLICA_COUNT`：TiFlash 副本数。
+`LOCATION_LABELS`：设置 TiFlash 副本时设置的 LocationLabelList。
+`AVAILABLE`：是否支持在 TiFlash 上做查询。如果为 1，TiDB 优化器将依据查询代价智能选择下推查询到 TiKV 或 TiFlash；如果为 0，TiDB 将不会下推查询到 TiFlash。
+`PROGRESS`：TiFlash 副本同步进度，精度为分钟级。如果 `PROGRESS` 小于 1，表示 TiFlash 副本落后 TiKV 较多，查询很可能会因为等待数据同步超时而失败。
+`TABLE_MODE`：TiFlash 是否开启 [FastScan](/develop/dev-guide-use-fastscan.md)。 Normal 表示禁用 FastScan，Fast 表示开启 FashScan。
```
Comment (qiancai) [inline information-schema/information-schema-tiflash-replica.md:39]:
```suggestion
`TABLE_MODE`：TiFlash 是否开启 [FastScan](/develop/dev-guide-use-fastscan.md)。 Normal 表示禁用 FastScan，Fast 表示开启 FastScan。
```

File: information-schema/information-schema-tiflash-replica.md [hunk 5]
```diff
@@ -29,3 +29,11 @@ DESC tiflash_replica;
...
 8 rows in set (0.01 sec)
 ```
+`TABLE_SCHEMA`：表所属的数据库的名称。
+`TABLE_NAME`：表的名称。
+`TABLE_ID`：表的内部 ID，该 ID 在一个 TiDB 集群内部唯一。
+`REPLICA_COUNT`：TiFlash 副本数。
+`LOCATION_LABELS`：设置 TiFlash 副本时设置的 LocationLabelList。
+`AVAILABLE`：是否支持在 TiFlash 上做查询。如果为 1，TiDB 优化器将依据查询代价智能选择下推查询到 TiKV 或 TiFlash；如果为 0，TiDB 将不会下推查询到 TiFlash。
+`PROGRESS`：TiFlash 副本同步进度，范围是[0,1]，精确到小数点后两位，刷新的精度为分钟级。如果 `PROGRESS` 小于 1，表示 TiFlash 副本落后 TiKV 较多，查询很可能会因为等待数据同步超时而失败。
+`TABLE_MODE`：TiFlash 是否开启 [FastScan](/develop/dev-guide-use-fastscan.md)。 Normal 表示禁用 FastScan，Fast 表示开启 FastScan。
```
Comment (JaySon-Huang) [inline information-schema/information-schema-tiflash-replica.md:39]: 这个应该从 6.3 版本后就移除掉了？ /cc @hongyunyan

Comment (hongyunyan) [reply information-schema/information-schema-tiflash-replica.md:39]: 是的，需要被移除掉

**Review by JaySon-Huang — APPROVED** [review]

File: information-schema/information-schema-tiflash-replica.md [hunk 6]
```diff
@@ -28,3 +28,10 @@ DESC tiflash_replica;
 +-----------------+-------------+------+------+---------+-------+
 7 rows in set (0.01 sec)
 ```
+`TABLE_SCHEMA`：表所属的数据库的名称。
+`TABLE_NAME`：表的名称。
+`TABLE_ID`：表的内部 ID，该 ID 在一个 TiDB 集群内部唯一。
+`REPLICA_COUNT`：TiFlash 副本数。
+`LOCATION_LABELS`：设置 TiFlash 副本时设置的 LocationLabelList。
+`AVAILABLE`：是否支持在 TiFlash 上做查询。如果为 1，TiDB 优化器将依据查询代价智能选择下推查询到 TiKV 或 TiFlash；如果为 0，TiDB 将不会下推查询到 TiFlash。
+`PROGRESS`：TiFlash 副本同步进度，范围是[0,1]，精确到小数点后两位，刷新的精度为分钟级。如果 `PROGRESS` 小于 1，表示 TiFlash 副本落后 TiKV 较多，查询很可能会因为等待数据同步超时而失败。
```
Comment (qiancai) [inline information-schema/information-schema-tiflash-replica.md:37]:
```suggestion
`PROGRESS`：TiFlash 副本同步进度，范围是 [0, 1]，精确到小数点后两位，刷新的精度为分钟级。如果 `PROGRESS` 小于 1，表示 TiFlash 副本落后 TiKV 较多，查询很可能会因为等待数据同步超时而失败。
```

File: information-schema/information-schema-tiflash-replica.md [hunk 7]
```diff
@@ -28,3 +28,10 @@ DESC tiflash_replica;
 +-----------------+-------------+------+------+---------+-------+
 7 rows in set (0.01 sec)
 ```
+`TABLE_SCHEMA`：表所属的数据库的名称。
```
Comment (qiancai) [inline information-schema/information-schema-tiflash-replica.md:31]:
```suggestion
`TIFLASH_REPLICA` 表中各列的字段含义如下：

`TABLE_SCHEMA`：表所属的数据库的名称。
```

File: information-schema/information-schema-tiflash-replica.md [hunk 8]
```diff
@@ -28,3 +28,12 @@ DESC tiflash_replica;
...
 7 rows in set (0.01 sec)
 ```
+`TIFLASH_REPLICA` 表中各列的字段含义如下：
+
+`TABLE_SCHEMA`：表所属的数据库的名称。
+`TABLE_NAME`：表的名称。
+`TABLE_ID`：表的内部 ID，该 ID 在一个 TiDB 集群内部唯一。
+`REPLICA_COUNT`：TiFlash 副本数。
+`LOCATION_LABELS`：设置 TiFlash 副本时设置的 LocationLabelList。
+`AVAILABLE`：是否支持在 TiFlash 上做查询。如果为 1，TiDB 优化器将依据查询代价智能选择下推查询到 TiKV 或 TiFlash；如果为 0，TiDB 将不会下推查询到 TiFlash。
```
Comment (qiancai) [inline information-schema/information-schema-tiflash-replica.md:38]:
```suggestion
`AVAILABLE`：表的 TiFlash 副本是否可用。1 代表可用，TiDB 优化器将依据查询代价智能选择下推查询到 TiKV 或 TiFlash；0 代表不可用，TiDB 将不会下推查询到 TiFlash。副本状态变为可用之后就不再改变，如果通过 DDL 命令修改副本数则会重新计算同步进度。
```

File: information-schema/information-schema-tiflash-replica.md [hunk 9]
```diff
@@ -28,3 +28,12 @@ DESC tiflash_replica;
...
+`PROGRESS`：TiFlash 副本同步进度，范围是 [0, 1]，精确到小数点后两位，刷新的精度为分钟级。如果 `PROGRESS` 小于 1，表示 TiFlash 副本落后 TiKV 较多，查询很可能会因为等待数据同步超时而失败。
```
Comment (qiancai) [inline information-schema/information-schema-tiflash-replica.md:39]:
```suggestion
`PROGRESS`：TiFlash 副本同步进度，范围是 [0, 1]，精确到小数点后两位，刷新的精度为分钟级。当 `AVAILABLE` 为 1 时，如果 `PROGRESS` 小于 1，表示 TiFlash 副本落后 TiKV 较多，下推到 TiFlash 的查询很可能会因为等待数据同步超时而失败。
```

File: information-schema/information-schema-tiflash-replica.md [hunk 10]
```diff
@@ -28,3 +28,12 @@ DESC tiflash_replica;
...
 ```
+`TIFLASH_REPLICA` 表中各列的字段含义如下：
+
+`TABLE_SCHEMA`：表所属的数据库的名称。
+`TABLE_NAME`：表的名称。
+`TABLE_ID`：表的内部 ID，该 ID 在一个 TiDB 集群内部唯一。
+`REPLICA_COUNT`：TiFlash 副本数。
+`LOCATION_LABELS`：设置 TiFlash 副本时设置的 LocationLabelList。
+`AVAILABLE`：表的 TiFlash 副本是否可用。1 代表可用，TiDB 优化器将依据查询代价智能选择下推查询到 TiKV 或 TiFlash；0 代表不可用，TiDB 将不会下推查询到 TiFlash。副本状态变为可用之后就不再改变。
+`PROGRESS`：TiFlash 副本同步进度，范围是 [0, 1]，精确到小数点后两位，刷新的精度为分钟级。当 `AVAILABLE` 字段为 1 时，如果 `PROGRESS` 小于 1，表示 TiFlash 副本落后 TiKV 较多，下推到 TiFlash 的查询很可能会因为等待数据同步超时而失败。
```
Comment (Oreoxmt) [inline information-schema/information-schema-tiflash-replica.md:39]:
```suggestion
`PROGRESS`：TiFlash 副本同步进度，范围是 `[0, 1]`，精确到小数点后两位，刷新的精度为分钟级。当 `AVAILABLE` 字段为 1 时，如果 `PROGRESS` 小于 1，表示 TiFlash 副本落后 TiKV 较多，下推到 TiFlash 的查询很可能会因为等待数据同步超时而失败。
```

File: information-schema/information-schema-tiflash-replica.md [hunk 11]
```diff
@@ -28,3 +28,12 @@ DESC tiflash_replica;
 +-----------------+-------------+------+------+---------+-------+
 7 rows in set (0.01 sec)
 ```
+`TIFLASH_REPLICA` 表中各列的字段含义如下：
```
Comment (qiancai) [inline information-schema/information-schema-tiflash-replica.md:31]:
```suggestion

`TIFLASH_REPLICA` 表中各列的字段含义如下：
```

File: information-schema/information-schema-tiflash-replica.md [hunk 12]
```diff
@@ -28,3 +28,13 @@ DESC tiflash_replica;
...
+
+`TIFLASH_REPLICA` 表中各列的字段含义如下：
+
+`TABLE_SCHEMA`：表所属的数据库的名称。
+`TABLE_NAME`：表的名称。
+`TABLE_ID`：表的内部 ID，该 ID 在一个 TiDB 集群内部唯一。
+`REPLICA_COUNT`：TiFlash 副本数。
+`LOCATION_LABELS`：设置 TiFlash 副本时设置的 LocationLabelList。
+`AVAILABLE`：表的 TiFlash 副本是否可用。1 代表可用，TiDB 优化器将依据查询代价智能选择下推查询到 TiKV 或 TiFlash；0 代表不可用，TiDB 将不会下推查询到 TiFlash。副本状态变为可用之后就不再改变。
+`PROGRESS`：TiFlash 副本同步进度，范围是 `[0, 1]`，精确到小数点后两位，刷新的精度为分钟级。当 `AVAILABLE` 字段为 1 时，如果 `PROGRESS` 小于 1，表示 TiFlash 副本落后 TiKV 较多，下推到 TiFlash 的查询很可能会因为等待数据同步超时而失败。
```
Comment (qiancai) [inline information-schema/information-schema-tiflash-replica.md:40]:
```suggestion
- `TABLE_SCHEMA`：表所属的数据库的名称。
- `TABLE_NAME`：表的名称。
- `TABLE_ID`：表的内部 ID，该 ID 在一个 TiDB 集群内部唯一。
- `REPLICA_COUNT`：TiFlash 副本数。
- `LOCATION_LABELS`：设置 TiFlash 副本时设置的 LocationLabelList。
- `AVAILABLE`：表的 TiFlash 副本是否可用。1 代表可用，TiDB 优化器将依据查询代价智能选择下推查询到 TiKV 或 TiFlash；0 代表不可用，TiDB 将不会下推查询到 TiFlash。副本状态变为可用之后就不再改变。
- `PROGRESS`：TiFlash 副本同步进度，范围是 `[0, 1]`，精确到小数点后两位，刷新的精度为分钟级。当 `AVAILABLE` 字段为 1 时，如果 `PROGRESS` 小于 1，表示 TiFlash 副本落后 TiKV 较多，下推到 TiFlash 的查询很可能会因为等待数据同步超时而失败。
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**